src/components/Alert/__tests__/index.test.tsx

```diff
@@ -17,8 +22,16 @@ describe('Alert', () => {
       shouldMatchEmotionSnapshot(<Alert variant={variant}>Sample Alert</Alert>))
   })
 
+  test(`renders correctly unknow variant and fallback to standard`, () =>
+    // @ts-ignore
```

Comment:
IMO, its because typescript compiler is crying on code editors 🤔 so we can't test unexpected case

Reply:
Codecov walks through this code: https://github.com/scaleway/scaleway-ui/blob/aca2ffb032f9b0230387ef3a3cc34afbd84fb428/src/components/Alert/index.tsx#L76
but before now, it will never pass through the second part (`variants(props).standard`). To do so, we must use a variant that isn't part of [AlertVariants](https://github.com/scaleway/scaleway-ui/blob/aca2ffb032f9b0230387ef3a3cc34afbd84fb428/src/components/Alert/index.tsx#L17) and so Typescript throws a type-error.

Reply:
Yes, I assumed, I meant to write it in the @ts-expect description. That way the next dev reading this code will understand :)

Reply:
Oh yes, you're right! Gonna add this tomorrow 🚀